totalopenstation/__init__.py

```diff
@@ -1,3 +1,13 @@
 # -*- coding: utf-8 -*-
 
 __version__  = '0.5.dev'
+
+import logging
+from pprint import pformat
+
+def geo_to_debug(data):
+    logger = logging.getLogger("tops")
```

Comment:
I'm not an expert in logging with Python, but according to [this well respected guide](https://docs.python-guide.org/writing/logging/) 

> It is strongly advised that you do not add any handlers other than NullHandler to your library’s loggers.

So if I understand correctly, only the 3 scripts must get the logger by name, and all other code should just follow the pattern explained in that guide. I have done some corrections in my local branch, perhaps I could push to your branch?

Reply:
Ok, lets go push it. ^^

Reply:
Function removed.
Use the guide for other commits...